engine/src/silo/whitelist.rs

```diff
@@ -10,9 +10,19 @@ const STATUS: &[u8] = b"LIST_STATUS";
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
 pub enum WhitelistKind {
+    /// The whitelist of this type is for storing NEAR accounts. Accounts stored in this whitelist
+    /// have an admin role. The admin role allows to add new admins and add new entities
+    /// (`AccountId` and `Address`) to whitelists. Also, this role allows to deploy of EVM code
+    /// and submit transactions.
     Admin = 0x0,
+    /// The whitelist of this type is for storing EVM addresses. Addresses included in this
+    /// whitelist can deploy EVM code.
```

Comment:
The description here is the same as for `Address`. What is the correct documentation?